### DISCUSSION TITLE
Viaplay does not work, F1TV is not available and slider does not work?

### DISCUSSION THREAD

**Discussion by raptile** [Nerwyn/universal-remote-card#109] [Q&A]
Hi,

Does anybody have the viaplay logo visible?
For me it does not work and if i press the empty space, viaplay does not start.

Also, does anybody have F1TV up and running?
- f1 is presented but when i click on it, notthing happens.

```
type: custom:android-tv-card
remote_id: remote.rapcast
slider_id: media_player.rapcast
rows:
  - - power
    - channel_up
    - info
    - channel_down
  - - appletv
    - f1
    - netflix
    - disney
    - primevideo
  - - viaplay
    - spotify
    - youtube
    - max
    - skyshowtime
  - - slider
  - - touchpad
  - - back
    - home
    - tv
  - - rewind
    - play_pause
    - fast_forward
```

Al the other buttons are working fine except the volume slider.

When i add the volume buttons, they work fine.
```
      - - volume_down
        - null
        - volume_mute
        - null
        - volume_up
```

<img width="505" alt="image" src="https://github.com/user-attachments/assets/c1a22013-a293-4520-99cb-b6d6712a6ba3">

Thanks in advance for the support!

**Comment by Nerwyn** [comment]
`viaplay` has an icon included but not a source for Android TV, as no one has provided one to add. And `f1` isn't a source, it's a function key like F1-F12.

**Reply by raptile** [reply]
Thanks for the answer to the F1 part :) I was afraid that would be it.

would it be possible to add viaplay and F1TV so those app will open?
How would i be able to give u what u need for these to work?

**Reply by Nerwyn** [reply]
Try following the [Android TV deep linking guide](https://community.home-assistant.io/t/android-tv-remote-app-links-deep-linking-guide/567921) and finding a custom SVG icon as [described in the README](https://github.com/Nerwyn/android-tv-card?tab=readme-ov-file#custom-icons).

**Reply by raptile** [reply]
I will give this a go when other projects have succeeded. thanks!
